suite-native/device/src/hooks/useDetectDeviceError.tsx

```diff
@@ -1,6 +1,9 @@
 import { useCallback, useEffect, useState } from 'react';
 import { useDispatch, useSelector } from 'react-redux';
 
+import * as Sentry from '@sentry/react-native';
```

Comment:
nitpick: I would prefer to import only used method 
```suggestion
import { captureException } from '@sentry/react-native';
```
but there is probably no real difference